docs/dev/app/supabase.md

```diff
@@ -2,4 +2,51 @@
 next: /dev/retool
 ---
 
+[[toc]]
```

Comment:
Could we remove the `[[toc]]`? It seems a bit redundant with the "On This Page" dropdown:
<img width="855" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/16c5f595-4978-4b2b-96fc-7fdd6f496ca2">

Or is there a way we can remove the dropdown? 🤔

Reply:
I'll remove the `[[toc]]`